scripts/common.sh

```diff
@@ -33,6 +33,79 @@ else
     function printLog() { echo "$(tput setaf 3)$1$(tput sgr0)"; }
 fi
 
+function fail() {
+    printError "$@"
+    return 1
```

Comment:
Why return instead of exit?

Reply:
Answered in https://github.com/ethereum/solidity/pull/12076/files#r721323664.